### DISCUSSION TITLE
基于微擎交付系统的一键部署方案，为社区贡献一份力量

### DISCUSSION THREAD

**Discussion by w7team** [GaiZhenbiao/ChuanhuChatGPT#997] [Show and tell]
### 可能的解决办法

【川虎Chat】为ChatGPT等多种LLM提供了一个轻快好用的Web图形界面和众多附加功能

安装地址：https://console.w7.cc/api/deploy/thirdparty_cd/redirect?route=/zpk-install?path=https%3A%2F%2Fzpk.w7.cc%2Frespo%2Finfo%2Fchatgpt_chuanhu

说明：安装地址为一键部署方案，服务器环境基于k3s，微擎交付系统是管理k3s的控制面板，可视化一键管理基于k3s部署的应用

### 帮助开发

- [ ] 我愿意协助开发！

**Comment by Keldos-Li** [comment]
我有点不确定你们是怎么进行版本控制的？会和我们的release tag同步吗？
<img width="937" alt="image" src="https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/23137268/28dffd22-30dd-4541-89ff-bc626c023eb8">

**Reply by w7team** [reply]
暂时还没支持tag同步，目前这个版本号是我们自定义的，用的是你们提供的docker镜像
